base_classes/nyaml/NXsample_component.yaml

```diff
@@ -128,7 +128,7 @@ NXsample_component(NXobject):
   (NXfabrication):
     doc: |
       Details about the sample component vendor (company or research group)
-  (NXsample_history):
+  sample_history(NXhistory):
```

Comment:
Maybe just history here? Or unnamed?

Reply:
I wanted it named because having mutliple histories doesn't make sense. Let's just call it `history(NXhistory)` everywhere (in instrument and, in the future, in the individual parts, too)?